.github/workflows/check-upstream-ed25519.yml

```diff
@@ -0,0 +1,46 @@
+on:
+  schedule:
+    - cron: '0 13 * * *'
+  workflow_dispatch:
+
+
+jobs:
+  check-ed25519-upstream:
+    name: Open an issue if upstream ed25519 has new commits
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check out repository
+      uses: actions/checkout@v2
```

Comment:
for those not familiar with Github actions: this uses-line refers to the Github repository "actions/checkout", where actions is an official Github organization -- so I'm not using any random action-repos here.